book/analysis/anagrams.py

```diff
@@ -80,48 +81,48 @@ def anagram_sort_and_compare(s1, s2):
 anagram_sort_and_compare('abcde', 'abcd')  # => False
 
 """
-At first glance you may be tempted to think that this algorithm is
-$$O(n)$$, since there is one simple iteration to compare the *n*
-characters after the sorting process. However, the two calls to the
-Python `sorted` method are not without their own cost. Sorting is
-typically either $$O(n^{2})$$ or $$O(n\log n)$$, so the sorting
-operations dominate the iteration. In the end, this algorithm will have
-the same order of magnitude as that of the sorting process.
+At first glance, you may be tempted to think that this algorithm is $$O(n)$$,
+since there is only one iteration to compare $$n$$ characters after sorting.
+However, the two `sorted` method calls have their own costs: sorting is
+typically either $$O(n^{2})$$ or $$O(n\log n)$$. Since both of these functions
```

Comment:
@aleksharma12 the n^2 implementations are rare enough that we could simplify this to "sorting is typically nlogn"